### PR TITLE
Add Nick to the bot, he spreads enough love. Time to receive it.

### PR DESCRIPTION
**Pull request by lukeocodes**
Meh.

TLDR: adds Nicholas Carrigan to the bot <3

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -34,6 +34,16 @@ const isLuke = (message: Message): boolean => {
   );
 };
 
+const isNick = (message: Message): boolean => {
+  return (
+    message.content.replace(/\s/g, "").toLowerCase().includes("nick") ||
+    message.content.replace(/\s/g, "").toLowerCase().includes("nicholas") ||
+    message.author.id === "465650873650118659" ||
+    message.content.includes("N I C K") ||
+    message.content.includes("N I C H O L A S")
```
Comment (MattIPv4) [inline src/index.ts:43]: These two lines achieve nothing not already achieved by the first two checks.

`"N I C K".replace(/\s/g, "").toLowerCase() => "nick"`

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (lukeocodes) [reply src/index.ts:43]: I thought that but I was too lazy to test it just copied his format 🙈

Comment (lukeocodes) [reply src/index.ts]: I've just seen what it was for. It was for the emoji match

Comment (MattIPv4) [reply src/index.ts]: Ah, these need to be regional indicator unicode chars then, not ascii :P

**Review by nhcarrigan — APPROVED** [review]
Feels weird to approve this 😛  but LGTM! 🥳

**Pull request merged**